### DISCUSSION TITLE
Link to both Coriolis and EDSY for ship loadout export

### DISCUSSION THREAD

**Discussion by Exsomnis78** [EDCD/EDMarketConnector#1031] [Ideas]
Instead of choosing between EDSY and Coriolis in the settings section, is it possible to add links to both in the "Ship:" section on EDMC main window so we can use both web sites simultaneously? Both sites have their own advantages. It may be shown as links such as "E" and "C" right to the ship's name. Thanks for the great tool, btw.

**Comment by Athanasius** [comment]
This is an excellent idea in principle.  The devil is in the details of the implementation.

I don't think I'd want to change the way the ship name is a hyperlink to the currently selected provider.  We *might* be able to make, e.g. holding ALT, cause use of the other provider.

Alternatively perhaps we could make tapping ALT whilst mouse-overing the hyperlink cycle through the providers (yes there are only two currently, but who knows what might be created in the future), which then requires some indication of the currently active one.

Or we could just have " (alt)" as a separate hyperlink on the end of the text to access the (currently) only other provider.

I'll go create an issue for this which will mean we'll be more likely to get around to it at some point.

**Reply by A-UNDERSCORE-D** [reply]
honestly a letter for each would be simple, or yeah alt would work well. looking to the future Im not sure how to implement this with new plugin stuff, as I intended to have providers like this be definable in any plugin. But thats future me's problem

**Reply by Athanasius** [reply]
I was mostly thinking that we don't want to *not* have the ship name be the link any more.  Users are used to clicking *it*.  So then if you have some "[c] [s]"  thing appended:

1. That's a single character to accurately click.
2. It says nothing about which one you'll get on the ship name click.

**Reply by Exsomnis78** [reply]
It may also be left click, right click or middle click for three options , without changing current view, also ship name would still be the link?
